resources/migrations/20191007092006-blacklist.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE blacklist_event (
+  id serial NOT NULL PRIMARY KEY,
+  eventdata jsonb
+);
+--;;
+CREATE INDEX index_blacklist_event_resource ON blacklist_event((eventdata->>'blacklist/resource'));
+--;;
+CREATE INDEX index_blacklist_event_user ON blacklist_event((eventdata->>'blacklist/user'));
```

Comment:
To serve queries where both resource and user is used in where,  an index which contains both of the fields would serve it better.

Reply:
e.g. one index for (resource, user) and anther index for (user)

Reply:
good point. I'll start with a (resource, user) index